build/ci-test-minikube.sh

```diff
@@ -1,24 +1,50 @@
-#!/usr/bin/env bash 
+#!/usr/bin/env bash
 
-set -xe
+set -x
 
-# Apply the required storage for minikube 
+function with_retries() {
```

Comment:
I might be confused but where is the loop?

Reply:
🤦‍♂️ I'll fix